src/sentry/apidocs/hooks.py

```diff
@@ -69,11 +72,64 @@ def __get_explicit_endpoints() -> List[Tuple[str, str, str, Any]]:
     ]
 
 
-def custom_preprocessing_hook(endpoints: Any) -> Any:  # TODO: organize method, rename
+def codemod_publish_status(endpoints):
```

Comment:
could we add a test for this? can refer to https://github.com/getsentry/sentry/blob/f8311c7198ba13488bd68174ba7a461d83b158c6/tests/sentry/apidocs/test_schema.py#L23

Reply:
@JoshFerge I don't plan to merge this. Run it just one time and then add code to build api so it fails if an api doesn't have this field for all methods.